src/vaadin-grid-pro-inline-editing-mixin.html

```diff
@@ -312,7 +316,16 @@
       }
     }
 
+    _setCancelCellSwitch() {
+      this.__cancelCellSwitch = true;
+    }
+
     _switchEditCell(e) {
+      if (this.__cancelCellSwitch) {
+        this.__cancelCellSwitch = false;
```

Comment:
Would it be safer to clean up asynchronously after the flag is set?

Reply:
Cleaned up asynchronously now. Added test case: `should be possible to switch edit cell...`